src/Microsoft.DotNet.ProjectJsonMigration/ProjectDependencyFinder.cs

```diff
@@ -347,7 +352,7 @@ public static bool TryGetGlobalSettings(string path, out GlobalSettings globalSe
                     var projects = settings["projects"];
                     var dependencies = settings["dependencies"] as JObject;
 
-                    globalSettings.ProjectPaths = projects == null ? new string[] { } : projects.Select(a => a.Value<string>()).ToArray();;
+                    globalSettings.ProjectPaths = projects == null ? new string[] { } : projects.Select(a => a.Value<string>()).ToArray(); ;
```

Comment:
You have an extra semi-column at the end of this line.

Also, NIT: break the line so that it fits within 120 characters. Makes it easier to read and find things like extra semi-columns.